Site/admin/components/Account/Merge.php

```diff
@@ -0,0 +1,264 @@
+		$ds = new SwatDetailsStore($account);
+		$ds->fullname = $account->getFullname();
+		return $ds;
+	}
+
+	// }}}
+	// {{{ protected function getAccount()
+
+	protected function getAccount()
+	{
+		if ($this->account === null) {
+			$account_class = SwatDBClassMap::get('SiteAccount');
+
+			$this->account = new $account_class();
+			$this->account->setDatabase($this->app->db);
+
+			if (!$this->account->load($this->id)) {
+				throw new AdminNotFoundException(sprintf(
+					Site::_('An account with an id of ‘%d’ does not exist.'),
+					$this->id
+				));
+			}
+
+			$instance_id = $this->app->getInstanceId();
+			if ($instance_id !== null) {
+				if ($this->account->instance->id !== $instance_id) {
+					throw new AdminNotFoundException(sprintf(
+						Store::_('Incorrect instance for account ‘%d’.'),
+						$this->id
+					));
+				}
+			}
+		}
+
+		return $this->account;
+	}
+
+	// }}}
+	// {{{ protected function buildAccountDetails()
+
+	protected function buildAccountDetails()
+	{
+		$ds = $this->getAccountDetailsStore();
+
+		$details_frame = $this->ui->getWidget('details_frame');
+		$details_frame->title = Site::_('Merge Account');
+		$details_frame->subtitle = $ds->fullname;
+
+		$details_view = $this->ui->getWidget('details_view');
+
+		$date_field = $details_view->getField('createdate');
+		$date_renderer = $date_field->getRendererByPosition();
+		$date_renderer->display_time_zone = $this->app->default_time_zone;
+
+		$details_view->data = $ds;
+	}
+
+	// }}}
+	// {{{ protected function buildNavBar()
+
+	protected function buildNavBar()
+	{
+		$this->navbar->addEntry(new SwatNavBarEntry(
+			$this->account->fullname,
+			sprintf('Account/Details?id=%s', $this->id)
+		));
+		$this->navbar->addEntry(new SwatNavBarEntry(Site::_('Merge')));
+		$this->title = Site::_('Merge');
+	}
+
+	// }}}
+	// {{{ protected function getTableModel()
+
+	protected function getTableModel(SwatView $view)
+	{
+		$search = $this->getAccountSearch();
+
+		$pager = $this->ui->getWidget('pager');
+		$pager->total_records = SwatDB::queryOne(
+			$this->app->db,
+			sprintf(
+				'select count(1) from Account %s where %s',
+				$search->getJoinClause(),
+				$this->getWhereClause()
+			)
+		);
+
+		$sql = sprintf(
+			$this->getSQL(),
+			$search->getJoinClause(),
+			$this->getWhereClause(),
+			$this->getOrderByClause($view, $search->getOrderByClause())
+		);
+
+		$this->app->db->setLimit($pager->page_size, $pager->current_record);
+
+		$accounts = SwatDB::query($this->app->db, $sql);
+
+		if (count($accounts) > 0) {
+			$this->ui->getWidget('results_message')->content =
+				$pager->getResultsMessage('result', 'results');
+		}
+
+		$class_name = SwatDBClassMap::get('SiteAccount');
+		$store = new SwatTableStore();
+		foreach ($accounts as $row) {
+			if ($row instanceof SiteAccount) {
+				$account = $row;
+			} else {
+				$account = new $class_name($row);
+				$account->setDatabase($this->app->db);
+			}
+			$store->add($this->getDetailsStore($account, $row));
```

Comment:
getDetailsStore is undefined. You should add an implementation that just returns
```php
new SwatDetailsStore($account);
```
subclasses can extend the method:
```
$ds = parent::getDetailsStore($account);
$ds->foo = $bar;
return $ds;
```